lib/karafka/contracts/consumer_group.rb

```diff
@@ -93,6 +94,10 @@ class ConsumerGroup < Dry::Validation::Contract
         end
       end
 
+      rule(:assignment_strategy) do
+        key.failure(:does_not_respond_to_call) if value && !value.respond_to?(:call)
```

Comment:
does nil or false respond to call? if not, then this is redundant and since as default we use roundrobin instance that responds to call, this seems redundant

Reply:
Good call, makes sense